torch/_inductor/codegen/cpp.py

```diff
@@ -1994,10 +2012,39 @@ def __init__(self, kernel_group):
 
     def legalize_bf16(self, nodes):
         def add_to_dtype(sub_graph: torch.fx.Graph):
+            def only_used_by_store(node: torch.fx.Node):
+                if len(node.users) > 1:
+                    return False
+                for user in node.users:
+                    _only_used_by_store = user.target == "store"
+                    opt_ctx = OptimizationContext()
+                    opt_ctx.is_directly_load_bf16 = _only_used_by_store
+                    node.meta[OptimizationContext.key] = opt_ctx
+                    return _only_used_by_store
```

Comment:
Is it fine regardless of how many users as long as the users are stores and the store dtype is bfloat16?

Reply:
Yes, thanks for advice.